src/app/components/Layout/Layout.jsx

```diff
@@ -1,7 +1,9 @@
-import React, { useContext, useState } from 'react'
+import React, { useContext, useEffect, useState } from 'react'
 import classNames from 'classnames'
 import { FaCog } from 'react-icons/fa'
-
+import {
+  VscChromeRestore, VscChromeMaximize, VscChromeMinimize, VscChromeClose
```

Comment:
Add newlines in between these

```suggestion
  VscChromeRestore,
  VscChromeMaximize,
  VscChromeMinimize,
  VscChromeClose
```